src/utils/utilities.jl

```diff
@@ -158,3 +158,45 @@ import ClimaCore
 import Krylov
 Krylov.ktypeof(x::ClimaCore.Fields.FieldVector) =
     ClimaComms.array_type(x){eltype(parent(x)), 1}
+
+
+using Printf
+# From BenchmarkTools
+function prettytime(t)
+    if t < 1e3
+        value, units = t, "ns"
+    elseif t < 1e6
+        value, units = t / 1e3, "μs"
+    elseif t < 1e9
+        value, units = t / 1e6, "ms"
+    else
+        value, units = t / 1e9, "s"
+    end
+    return string(@sprintf("%.3f", value), " ", units)
+end
+
+function prettymemory(b)
+    if b < 1024
+        return string(b, " bytes")
+    elseif b < 1024^2
+        value, units = b / 1024, "KiB"
+    elseif b < 1024^3
+        value, units = b / 1024^2, "MiB"
+    else
+        value, units = b / 1024^3, "GiB"
+    end
+    return string(@sprintf("%.2f", value), " ", units)
```

Comment:
```suggestion
    return @sprintf("%.2f %s", value, units)
```

Reply:
I'm happy to make these changes, but I am curious, what's the benefit?